### PR TITLE
MAINT: Update backronym

### PR DESCRIPTION
**Pull request by ElDeveloper**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2016, Yoshiki Vázquez Baeza
+Copyright (c) 2016, The LOL Developers
```
Comment (SantiagoTorres) [inline LICENSE:1]: LOL

**Comment by SantiagoTorres** [comment]
Looks good! :+1:

**Comment by ElDeveloper** [comment]
If you are good with this, would you mind merging this branch?

**Comment by SantiagoTorres** [comment]
Done

**Pull request merged**